lib/nats.js

```diff
@@ -876,6 +876,17 @@ Client.prototype.processMsg = function() {
 };
 
 /**
+ * Push a new cluster server.
+ *
+ * @param {String} uri
+ * @api public
+*/
+
+Client.prototype.addServer = function(uri) {
+  this.servers.push(new Server(url.parse(uri)));
```

Comment:
I assumed you're leaving this to the developer since there are currently no checks in the constructor; but I could add checks. If server already exists, ignore. Invalid uri, throw error.